source/features/releases-tab.tsx

```diff
@@ -10,9 +10,9 @@ import * as api from '../github-helpers/api';
 import looseParseInt from '../helpers/loose-parse-int';
 import {appendBefore} from '../helpers/dom-utils';
 import {createDropdownItem} from './more-dropdown';
-import {buildRepoURL, getRepoURL, getRepoGQL} from '../github-helpers';
+import {buildRepoURL, getRepoGQL, getRepositoryInfo} from '../github-helpers';
 
-const cacheKey = `releases-count:${getRepoURL()}`;
+const cacheKey = `releases-count:${getRepositoryInfo()!.url}`;
```

Comment:
Most of the changes look like this. Given that `getRepositoryInfo` is now normalized via meta tag, capitalization should not be a problem.